content/post/2022-09-01-r-dependency/index.Rmd

```diff
@@ -0,0 +1,223 @@
+---
+slug: r-dependency
+title: "Checking the inputs of your R functions" 
+authors: 
+- Hugo Gruson
+- Maëlle Salmon
```

Comment:
```suggestion
```

Reply:
unless I provide really substantial input later on, this is your own work :wink:

Reply:
I still have a copy of our initial conversation and I can confirm you contributed quite a lot to what is summarised in https://github.com/r-hub/blog/issues/140#issuecomment-1064323165.

Reply:
Then my memory didn't keep this copy :joy: Thank you!